foundation_cms/static/scss/buttons.scss

```diff
@@ -266,3 +266,25 @@ button.btn-play--vimeo {
     }
   }
 }
+
+// ===================================
+// Topic Link Buttons
+// ===================================
+
+@mixin topic-pill-button-shape {
```

Comment:
I see that this shape will be used elsewhere on the site. Thought it would make sense to create a mixin for it.